_layouts/default.new.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+
+<html lang="{{ page.lang | default: site.lang | default: "es" }}">
+  {% include head.new.html %}
+  <body>
+    <div class="container mx-auto m-10">
+      <div class="flex flex-row justify-between">
+        <a href="/">
+          <img src='{{ "/assets/img/logo.png" | prepend:site.baseurl }}' />
+        </a>
+
+        <nav class="flex">
+          <ul class="flex text-gray-300">
+            <li class="ml-8"><a href={{ site.baseurl }}>{% t menu.home %}</a></li>
+            <li class="ml-8"><a href={{ "/blog.html" | prepend: site.baseurl }}>{% t menu.blog %}</a></li>
+          </ul>
+
+          {% include language_switcher.new.html %}
+        </div>
+
+        {% if page.with_brand %}
+          {% include navbar_brand.html %}
+        {% endif %}
+      </div>
+
+      <div class="mb-48 mt-64">
+        <div class="max-w-3xl mx-auto mb-16">
+          <h1 class="font-sans font-extrabold text-5xl text-gray-700 leading-tight max-w-lg mb-16">Software lliure per a l'economia social</h1>
+          <p class="font-sans text-2xl text-gray-300 leading-relaxed mb-16">Coopdevs és una associació de persones que té com a objectiu <strong class="text-gray-700">impulsar solucions tecnològiques per fomentar i promoure l’economia social.</strong> Tots els projectes en què participem són de codi obert i llicència lliure.</p>
```

Comment:
Nice update!

>  I know it's not fair to only show Coopdevs SCCL members in the pictures

I think it's impossible and unnecessary to add everyone in the association. If we could one picture of all that would be great, but since there is also a SCCL doing actual work, I am ok with it being represented here (unless you want your own website). In fact, you may want to consider changing the "Coopdevs és una associació" to "Coopdevs és una cooperativa"…

Reply:
> In fact, you may want to consider changing the "Coopdevs és una associació" to "Coopdevs és una cooperativa"

That feels too much but thanks for the feedback. I won't feel that bad from now on :joy: